cmd/root.go

```diff
@@ -49,27 +49,29 @@ var (
 	executionWaitGroup sync.WaitGroup
 )
 
-// RootCmd represents the base command when called without any subcommands.
-var RootCmd = &cobra.Command{
-	Use:   "bbi",
-	Short: "manage and execute models",
-	Long:  fmt.Sprintf("bbi CLI version %s", VERSION),
-}
 
 // Execute adds all child commands to the root command sets flags appropriately.
 // This is called by main.main(). It only needs to happen once to the rootCmd.
 func Execute(build string) {
 	if build != "" {
 		VERSION = fmt.Sprintf("%s-%s", VERSION, build)
 	}
-	RootCmd.Long = fmt.Sprintf("bbi cli version %s", VERSION)
-	if err := RootCmd.Execute(); err != nil {
+	cmd := NewRootCmd()
+	cmd.Long = fmt.Sprintf("bbi cli version %s", VERSION)
+	if err := cmd.Execute(); err != nil {
 		fmt.Println(err)
 		os.Exit(-1)
 	}
 }
 
-func init() {
+// RootCmd represents the base command when called without any subcommands.
+func NewRootCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "bbi",
+		Short: "manage and execute models",
+		Long:  fmt.Sprintf("bbi CLI version %s", VERSION),
+	}
+
 	// Set random for application
 	rand.Seed(time.Now().UnixNano())
```

Comment:
I'd like to know what is up with this reset of the seed at some point, but It's fine for now.